packages/mux-player-react/package.json

```diff
@@ -28,10 +35,14 @@
     "clean": "shx rm -rf dist/",
     "dev:cjs": "yarn build:cjs --watch=forever",
     "dev:esm": "yarn build:esm --watch=forever",
+    "dev:cjs:lazy": "echo 'esbuild cjs does not support code-splitting. See https://esbuild.github.io/api/#splitting for details'",
+    "dev:esm:lazy": "yarn build:esm:lazy --watch=forever",
     "dev:types": "yarn build:types -w",
-    "dev": "npm-run-all --parallel dev:types dev:cjs dev:esm",
+    "dev": "npm-run-all --parallel dev:types dev:cjs dev:esm dev:esm:lazy",
     "build:cjs": "esbuild src/index.tsx --target=es2019 --bundle --sourcemap --metafile=./dist/cjs.json --format=cjs --loader:.css=text --outdir=dist --out-extension:.js=.cjs.js --external:react --external:@mux/* --external:prop-types --define:PLAYER_VERSION=\"'$npm_package_version'\"",
     "build:esm": "esbuild src/index.tsx --target=es2019 --bundle --sourcemap --metafile=./dist/esm.json --format=esm --loader:.css=text --outdir=dist --out-extension:.js=.mjs --external:react --external:@mux/* --external:prop-types --define:PLAYER_VERSION=\"'$npm_package_version'\"",
+    "build:cjs:lazy": "echo 'esbuild cjs does not support code-splitting. See https://esbuild.github.io/api/#splitting for details'",
+    "build:esm:lazy": "esbuild src/lazy.tsx --splitting --target=es2019 --bundle --sourcemap --metafile=./dist/esm.lazy.json --format=esm --loader:.css=text --outdir=dist --out-extension:.js=.mjs --external:react --external:@mux/* --external:prop-types --define:PLAYER_VERSION=\"'$npm_package_version'\"",
```

Comment:
this probably needs to get added to the `build` step. Also, do types get generated for it?

Reply:
I don't think so, because... something something turborepo?
https://github.com/muxinc/elements/pull/414#issuecomment-1267538556
@luwes

Reply:
`build:types` is called by Turborepo so it's not needed to add to the list of the `build` command but in the command `build:types` there should probably be something added for the lazy package as well

e.g. "build:types": "tsc normalplayer && tsc lazyplayer"

Reply:
Thanks for clarifying! I'm on it.

Reply:
But does anything build `build:esm:lazy`?

Reply:
@luwes and @gkatsev sat down to do the thing and here's the output of `build:types` with no changes. Am I understanding correctly that tsc already knows about `lazy` and no action is required here?
<img width="269" alt="image" src="https://user-images.githubusercontent.com/8933386/194938506-4dd8c0a3-0bd6-4680-9608-d701b722dcd2.png">

Reply:
yup you're right it's good already. sorry about that